openreview/venue_request/process/recruitmentProcess.py

```diff
@@ -48,6 +46,7 @@ def process(client, note, invitation):
         reviewers_name = role_name,
         title = note.content['invitation_email_subject'].strip(),
         message = note.content['invitation_email_content'].strip(),
+        reduced_load_on_decline = reduced_load,
```

Comment:
what happens if the PCs don't want to allow reduced loads? is `reduced_load` an empty array or None?

Reply:
can you remove the string `reviewers_only` here: https://github.com/openreview/openreview-py/blob/master/openreview/venue_request/venue_request.py#L981

Also I'm not sure if we should define a default value, I think we should make this optional. If they want to enable reduced quotas then they should enter the comma separated values.

Reply:
Could you also show the reduced load in the AC console? You can re-use the same code that we use in the Reviewers console:

https://github.com/openreview/openreview-py/blob/master/openreview/conference/templates/reviewerWebfield.js#L247

First we try to check if there is already a Custom_Max_Papers edge and show it. If there is not edge created yet then we check if there is a Reduced_Load note and show the latest one.